homepage/home.go

```diff
@@ -0,0 +1,47 @@
+package homepage
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/planetdecred/pdanalytics/attackcost"
+	"github.com/planetdecred/pdanalytics/web"
+)
+
+type Home struct {
+	server *web.Server
+	mods   Mods
+}
+
+type Mods struct {
```

Comment:
What's this for?

Reply:
For `Home` to interface with the other mods. Updated the pr with a basic implementation.